ofctrl/ofSwitch.go

```diff
@@ -493,7 +486,7 @@ func (self *OFSwitch) unSubscribeMessage(xID uint32) {
 
 func (self *OFSwitch) sendModPortMessage(port int, mac net.HardwareAddr, config int, mask int) error {
 	msg := openflow15.NewPortMod(port)
-	msg.Header.Version = 0x4
+	msg.Header.Version = 0x6
```

Comment:
Is this fixing a bug?

Reply:
Yes, it is a bug fix, and I would mention it in the commit message.